image/templates/helm/shared/config-templates/scanner-v4/indexer-config.yaml.tpl

```diff
@@ -21,6 +21,8 @@ indexer:
       client_encoding=UTF8
     password_file: /run/secrets/stackrox.io/secrets/password
   get_layer_timeout: 1m
```

Comment:
@mclasmeier 
I am wondering if we could make this config to be conditional:
for indexer in secured cluster, we want to use sensor endpoint: `https://sensor.stackrox.svc/scanner/definitions`
for indexer in central cluster, we want to use central endpoint: `https://central.stackrox.svc/api/extensions/scannerdefinitions`

Reply:
Hi! See https://github.com/stackrox/stackrox/pull/9225/files